test/runtests.jl

```diff
@@ -45,10 +45,10 @@ end
 
 A = [1 0; 0 -2]
 F = eigfact(Positive, A)
-if VERSION < v"v0.5-"
-    @test_approx_eq full(F) abs(A)
-else
+if isdefined(:≈)
     @test full(F) ≈ abs.(A)
```

Comment:
I didn't do this originally, due to https://github.com/JuliaLang/Compat.jl/issues/310. Fixed now